src/StringTools/InternableString.cs

```diff
@@ -96,7 +96,7 @@ public bool MoveNext()
         /// </summary>
         private readonly ReadOnlySpan<char> _inlineSpan;
 
-#if NETSTANDARD
+#if RUNTIME_TYPE_NETCORE
```

Comment:
Can you add a feature flag that's turned on in our core builds only that's like `FEATURE_FASTSPAN` and switch these to

```suggestion
#if FEATURE_FASTSPAN
```